src/settings/MappingProfiles/MappingProfilesTransformationsModal/MappingProfilesTransformationsModal.js

```diff
@@ -0,0 +1,177 @@
+import React, {
+  useState,
+  useEffect,
+} from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { noop } from 'lodash';
+
+import {
+  Button,
+  Icon,
+  Modal,
+  Pane,
+  Paneset,
+  SearchField,
+  AccordionSet,
+  Accordion,
+  FilterAccordionHeader,
+  Checkbox,
+} from '@folio/stripes/components';
+import {
+  ExpandFilterPaneButton,
+  CollapseFilterPaneButton,
+} from '@folio/stripes/smart-components';
+
+import { TransformationsForm } from './TransformationsForm';
+import { mappingProfileTransformations } from './TransformationsField/transformations';
+
+import css from './MappingProfilesTransformationsModal.css';
+
+export const MappingProfilesTransformationsModal = ({
+  isOpen,
+  initialTransformationsValues,
+  onCancel,
+}) => {
+  const [isFilterPaneOpen, setFilterPaneOpen] = useState(true);
+
+  const totalSelectedCount = 0;
+  const searchResultsCount = mappingProfileTransformations.length;
+
+  useEffect(() => {
```

Comment:
Please, could you clarify why we need it here?

Reply:
In case the filter pane is closed, and then the modal is closed and then after its reopening the filter remained closed. Effect resets the filter pane to the opened state whenever the modal is closed.